marcel.py

```diff
@@ -46,14 +46,14 @@
 
 def check_for_marcefile(command):
     """
-    Detect if a RecettesAMarcel file is present in the current directory.
-    If so, inject a "-f ./RecettesAMarcel" argument in the docker build command,
+    Detect if a RecettesÀMarcel file is present in the current directory.
+    If so, inject a "-f ./RecettesÀMarcel" argument in the docker build command,
     if such an argument was not already passed.
     """
-    if exists(join(dirname(__file__), 'RecetteAMarcel')):
+    if exists(join(dirname(__file__), 'RecetteÀMarcel')):
```

Comment:
Please use ``u`` prefix, as it contains non ascii characters. We'd like to support python 2 too.